community_governance.adoc

```diff
@@ -261,10 +257,11 @@ Do not expect to influence decisions in a do-ocracy until you are able to demons
 The founder-leader governance model is most common among new projects or those with a small number of contributors (and since most open source projects have only a small number of contributors, this is a rather popular model!).
 In these projects, the individual or group who started the project also administers the project, establishes its vision, controls all permissions to merge code into it, and assumes the right to speak for it in public.
 Some projects refer to their founder-leaders as "BDFLs" or "Benevolent Dictators for Life."
+// @quaid I thought we weren't going to make the BDFL reference? Maybe reference it as an archaic form falling out of favor in recent years, etc.?
```

Comment:
We decided not to omit entirely, but to relegate to the description of the governance model, not make it the label for this model/section. We might even consider specifying further in the language here (something like "... for Life," a term that is falling out of fashion.")

Reply:
I like the falling out of fashion option.